### PR TITLE
fix: render mathjax formulas within glossary terms & tablepress tables

### PR DESCRIPTION
**Pull request by richard015ar**
Issue: https://github.com/pressbooks/pressbooks/issues/3408

This PR allows the `latex` shortcode to be rendered in the Glossary Term posts type (web, EPUB and PDF versions) as well as within the glossary term modal in the web version.
In addition, it also allows latex formulas to be properly rendered within TablePress tables.

### Test case
- Create one or more glossary terms using the latex shortcode, for example: `This is a glossary term with a math expression: [latex]x ^(3y) - 9[/latex]`.
- Add 1 or more TablesPress tables with latex formulas in it.
- Use this glossary in one or more chapters using the glossary shortcode. For example, in a post content: `I here's the [glossary id="<YOUR GLOSSARY ID HERE>"]term[/glossary].`.
- Create a Back Matter with `Glossary` Post type. Leave it blank so it can be populated with your list of glossary terms.
- Go to the web version of your posts containing the glossary terms. Clicking on the terms, you should be able to see the glossary pop-up with the mathjax formulas properly rendered as well as the tablepress table with formulas.
- Visit the Back Matter for the list of glossary terms. You should be able to see the mathjax formulas properly rendered for those glossary terms containing mathjax formulas.
- Export the book to PDF and EPUB. You should be able to see the glossary terms with the formulas as expected in the back matter post type.

### CODE REVIEW

File: assets/dist/mix-manifest.json [hunk 1]
```diff
@@ -11,11 +11,11 @@
     "/scripts/ftnref-convert.js": "/scripts/ftnref-convert.js?id=818f4c429292cb7b6f34057dfcff8b09",
     "/scripts/latex.js": "/scripts/latex.js?id=6393d17f0cdf9131ef0569f5801f1977",
     "/scripts/glossary.js": "/scripts/glossary.js?id=2ff3ebc6fb84ebb5d7c6009d05fcc102",
-    "/scripts/glossary-definition.js": "/scripts/glossary-definition.js?id=e6303e5bdde3c657763391c81f9be4f6",
+    "/scripts/glossary-definition.js": "/scripts/glossary-definition.js?id=1b15b88ec76ae81f1dffb51b9a94d1b1",
     "/scripts/import.js": "/scripts/import.js?id=212e58dacae764405a4408ce7e20c44d",
     "/scripts/login.js": "/scripts/login.js?id=428e5945062c207eabfb8a071122b059",
     "/scripts/network-managers.js": "/scripts/network-managers.js?id=c5acf0a3a6f43e00081189e85a8176e0",
-    "/scripts/organize.js": "/scripts/organize.js?id=22b1775082cd8db5cf4892595eed526c",
+    "/scripts/organize.js": "/scripts/organize.js?id=e49e282f09d8ad872683b175a17f3d37",
```
Comment (richard015ar) [inline assets/dist/mix-manifest.json]: It is strange that changing a single line in a js produced so many changes in other js files after compilation using `npm run build`.
It seems like we do not run this command for a long time and we did not merge this to `dev`. I am not sure if this is the case.

File: inc/shortcodes/glossary/class-glossary.php [hunk 2]
```diff
@@ -294,7 +292,7 @@ function tooltipContent( $content ) {
 
 		foreach ( $glossary_terms as $glossary_term_id => $glossary_term ) {
 			$identifier = 'term_' . $id . '_' . $glossary_term_id;
-			$content .= '<template id="' . $identifier . '"><div class="glossary__definition" role="dialog" data-id="' . $identifier . '"><div tabindex="-1">' . wpautop( do_shortcode( $glossary_term ) ) . '</div><button><span aria-hidden="true">&times;</span><span class="screen-reader-text">' . __( 'Close definition', 'pressbooks' ) . '</span></button></div></template>';
+			$content .= '<template id="' . $identifier . '"><div class="glossary__definition" role="dialog" data-id="' . $identifier . '"><div tabindex="-1">' . wpautop( apply_shortcodes( $glossary_term ) ) . '</div><button><span aria-hidden="true">&times;</span><span class="screen-reader-text">' . __( 'Close definition', 'pressbooks' ) . '</span></button></div></template>';
```
Comment (richard015ar) [inline inc/shortcodes/glossary/class-glossary.php:295]: This is just an alias. I just replaced it, to do a tiny step towards: https://github.com/pressbooks/pressbooks/issues/2183

File: assets/dist/mix-manifest.json (hunk 1 above)
Comment (fdalcin) [reply assets/dist/mix-manifest.json]: This was happening due to an old version of `node`. Fixed after upgrading it.

**Review by fdalcin — APPROVED** [review]
LGTM, changes work as expected locally.

**Pull request merged**